DigitalLearningSolutions.Web/Views/SuperAdmin/Faqs/SuperAdminFaqs.cshtml

```diff
@@ -0,0 +1,48 @@
+@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Faqs
+@using Microsoft.Extensions.Configuration
+@model FaqsPageViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/superAdmin/faqs.css")" asp-append-version="true">
+
+@section scripts {
+  <script src="@Url.Content("~/js/support/faqs.js")" asp-append-version="true"></script>
```

Comment:
Note that this `JS` file has a check built in that will mean it will not work for `SuperAdmin` as it stands. 
The url defined in there includes `/Support/` which will not match the url here.